js/civs.js

```diff
@@ -1,18 +1,199 @@
-function civ(name, tree) {
-    resetToDefault(tree);
+function parseCiv(civ) {
+   let builder = new CivBuilder({uniqueUnit: civ.unique[0],
+         uniqueUnitElite: civ.unique[1],
+         uniqueTechOne: civ.unique[2],
+         uniqueTechTwo: civ.unique[3],
+         monkPrefix: civ.monkPrefix});
+   
+   // Disable bldgs, units, techs.
+   if (civ.disabled) {
+      if (civ.disabled.buildings) builder.disableBuildings(civ.disabled.buildings);
+      if (civ.disabled.units) builder.disableUnits(civ.disabled.units);
+      if (civ.disabled.techs) builder.disableTechs(civ.disabled.techs);
+   }
 
-    let selectedCiv = civsConfig[name];
+   // Enable bldgs, units, techs.
+   if (civ.enabled) {
+      if (civ.enabled.buildings) builder.enableBuildings(civ.enabled.buildings);
+      if (civ.enabled.units) builder.enableUnits(civ.enabled.units);
+      if (civ.enabled.techs) builder.enableTechs(civ.enabled.techs);
```

Comment:
I know this repository does not have an explicit style guide, but please do not use if statements without curly brackets.